logparser/devices/outputdevices.py

```diff
@@ -64,7 +64,14 @@ def write(self, text=""):
         if self.support_ansi:
             print("\033[K" + text)
         else:
-            print(text)
+            # Catch any potential exception when piping the output and
+            # terminating the program.
+            try:
+                print(text)
+            except IOError:
+                # It makes no sense to print the error since we already had
```

Comment:
The exception happens in the native `print` operation because the pipe is closed.